dart/lib/src/protocol/sentry_device.dart

```diff
@@ -221,27 +221,14 @@ class SentryDevice {
 
   /// Produces a [Map] that can be serialized to JSON.
   Map<String, dynamic> toJson() {
-    String? orientation;
-
-    switch (this.orientation) {
-      case SentryOrientation.portrait:
-        orientation = 'portait';
-        break;
-      case SentryOrientation.landscape:
-        orientation = 'landscape';
-        break;
-      case null:
-        orientation = null;
-        break;
-    }
     return <String, dynamic>{
       if (name != null) 'name': name,
       if (family != null) 'family': family,
       if (model != null) 'model': model,
       if (modelId != null) 'model_id': modelId,
       if (arch != null) 'arch': arch,
       if (batteryLevel != null) 'battery_level': batteryLevel,
-      if (orientation != null) 'orientation': orientation,
+      if (orientation != null) 'orientation': orientation!.name,
```

Comment:
Can't this result in a force unwrap exception?

Reply:
Don't think so because `orientation` is `final`, and we do `orientation != null` before.
I also don't think `!` but if it's `final`, it's safe.

Reply:
> and we do orientation != null before.

True, I missed that.